### DISCUSSION TITLE
İBAX Cüzdan Bağlama

### DISCUSSION THREAD

**Discussion by coinsinov** [IBAX-io/go-ibax#872] [Soru Cevap - Türkçe]
İBAX Cüzdan oluşturduk. Var olan Coinleri Cüzdana Bağlama olayını resimli anlatım yapmanız mümkün mü? Teşekkür ederim.

**Comment by Yusufknk** [comment]
Cüzdan baglama işlemi My dashboard-retwet bölümünde alt tarafta gösterilen videodan yapabilirsiniz.(Cüzdanı bağlandıktan sonra lütfen private keyleri alınız.) Puanların karşılığı 1 Şubat'tan sonra cüzdanlara gönderilecek.

**Reply by coinsinov** [reply]
Bunları yaptım Cüzdan adresim var. Get IBAX mneüsünden baktığımda Cüzdan Adresimin altında Depolama Bloğu numarası ve altında 
View Blocks Information menüsü var. Orada bir şey yapmam gerekiyor mu Yusuf Hocam?

**Comment by Yusufknk** [comment]
Hayır, 1 Şubat'ı bekleyin.

**Reply by coinsinov** [reply]
Tamamdır teşekkür ederim Yusuf Hocam.

**Reply by Garadepelim** [reply]
Teşekkürler hocam. Bugünki Retweet görevinde yazanlardan sonra biraz kafa karışıklığı olmuştu.
